File: backend/app/Models/Comment.php
<?php

namespace App\Models;

use Eloquent;
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Database\Eloquent\Concerns\HasAttributes;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

/**
 * @mixin Eloquent
 * @mixin Builder
 */
class Comment extends Model
{
  use HasAttributes;

  protected $fillable = [
    'text',
    'user_id',
    'article_id',
  ];

  public function user(): BelongsTo
  {
    return $this->belongsTo(User::class, 'author_id');
  }

  public function article(): BelongsTo
  {
    return $this->belongsTo(Article::class);
  }
}

File: UserPanel/Models/Comment.cs
using System.Text.Json.Serialization;
namespace UserPanel.Models;
public class Comment
{
    [JsonPropertyName("id")] public required int Id { get; set; }
    [JsonPropertyName("text")] public required string Text { get; set; }
    [JsonPropertyName("user_id")] public required int UserId { get; set; }
    [JsonPropertyName("article_id")] public required int ArticleId { get; set; }
}